x-pack/plugins/security_solution/server/lib/detection_engine/prebuilt_rules/model/rule_assets/prebuilt_rule_asset.ts

```diff
@@ -29,10 +29,15 @@ import {
  *  - version is a required field that must exist
  */
 export type PrebuiltRuleAsset = z.infer<typeof PrebuiltRuleAsset>;
-export const PrebuiltRuleAsset = BaseCreateProps.and(TypeSpecificCreateProps).and(
+export const PrebuiltRuleAsset = BaseCreateProps.merge(
   z.object({
-    rule_id: RuleSignatureId,
-    version: RuleVersion,
     required_fields: RequiredFieldArray.optional(),
```

Comment:
Isn't `required_fields` already a part of `BaseCreateProps`?

Reply:
As discussed offline, we recalculate the `ecs` field when installing rules, so its presence in the rule assets doesn't affect the process. For simplicity, we can omit it.